### PR TITLE
pscanrulesBeta: add website links to help pages

### PR DESCRIPTION
**Pull request by PariharAditya**
## Overview
Implemented a new interface for linking to website
updated help with there respective ID

## Related Issues
https://github.com/zaproxy/zaproxy/issues/8189

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesBeta;
+
+public interface CommonPassiveScanRuleBeta {
+    public int getPluginId();
+
+    public default String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/passive-scan-rules-alpha/#id-"
```
Comment (kingthorin) [inline addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java:26]: beta

File: addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java
Comment (kingthorin) [inline addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java:1]: CommonPassiveScanRuleInfo

It'll be package specific, but that'll keep life easier as rules are promoted (if it's a consistent name).

File: addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public interface CommonPassiveScanRuleBeta {
     public int getPluginId();
 
     public default String getHelpLink() {
-        return "https://www.zaproxy.org/docs/desktop/addons/passive-scan-rules-alpha/#id-"
+        return "https://www.zaproxy.org/docs/desktop/addons/passive-scan-rules-alpha-beta/#id-"
```
Comment (kingthorin) [inline addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java:26]: Just beta :)

https://www.zaproxy.org/docs/desktop/addons/passive-scan-rules-beta/

Comment (PariharAditya) [reply addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java:26]: oh, sorry

File: addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java
Comment (PariharAditya) [reply addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/CommonPassiveScanRuleBeta.java:1]: ok

**Comment by kingthorin** [comment]
I'm not sure why you've added a commit from another account, you can do an interactive rebase and remove that.
For the original set of commits it seems your GitHub account isn't associated with the email you used in your git info (which you can see here: https://github.com/zaproxy/zap-extensions/commit/7d21d3ca6a3a35a3b2a1c7652abd2b3b8481ded2.patch)

**Pull request closed without merging**